bazarr/app/config.py

```diff
@@ -8,6 +8,8 @@
 import secrets
 import threading
 import time
+from datetime import datetime
+import random
```

Comment:
These imports are for user-friendly date and time with randomness when it was migrated.